### DISCUSSION TITLE
Does Milvus 2.4.0-rc.1 support nvidia jetson?

### DISCUSSION THREAD

**Discussion by sanujenLLM** [milvus-io/milvus#33595] [Q&A and General discussion]
Does Milvus 2.4.0-rc.1 support nvidia jetson?

**Comment by yhmo** [comment]
nvidia jetson is arm os. milvus has arm-based docker image, but I don't think it can directly run on nvidia jetson.

**Reply by sanujenLLM** [reply]
https://milvus.io/docs/index-with-gpu.md

If it can run, Will the GPU indexing be possible there?

**Reply by yhmo** [reply]
GPU index mainly depends GPU compute capability.
The compute capability of your GPU device must be one of the following: 6.0, 7.0, 7.5, 8.0, 8.6, 9.0. 
https://milvus.io/docs/prerequisite-gpu.md

**Reply by sanujenLLM** [reply]
![image](https://github.com/milvus-io/milvus/assets/153171691/6d8440d5-43dc-4a16-bd87-7ce6d796867d)

I got this for jetson AGX devices and 8.7 is the mentioned compute capability is that fine?

**Reply by yhmo** [reply]
Jetson Nano is 5.3, cannot meet the requirement.
Other Jetson devices probably meet the requirement, but it is risky because 6.2/7.2/8.7 might not compatible with 6.0/7.0/8.6
